### ISSUE TITLE
`MultipleAugmentationEntriesError` not imported

### ISSUE THREAD

**Issue by rtweeks** [PayTrace/intercom_test#14]
```
  File "/home/xxxxxx/pyenv/lib/python3.9/site-packages/intercom_test/augmentation/update_file.py", line 201, in index
    raise MultipleAugmentationEntriesError(
NameError: name 'MultipleAugmentationEntriesError' is not defined
```

**Comment by rtweeks** [comment]
I'm working on a patch for this.

**Comment by rtweeks** [comment]
Fixed in v2.3.0
